src/blocks/bluetooth.rs

```diff
@@ -224,27 +224,32 @@ pub struct Bluetooth {
     format_unavailable: FormatTemplate,
 }
 
-#[derive(Deserialize, Debug, Clone)]
-#[serde(deny_unknown_fields, default)]
+#[derive(Deserialize, Debug, Default, Clone)]
+#[serde(deny_unknown_fields)]
 pub struct BluetoothConfig {
     pub mac: String,
     //DEPRECATED
     //TODO remove
     pub label: Option<String>,
+    #[serde(default = "BluetoothConfig::default_hide_disconnected")]
     pub hide_disconnected: bool,
+    #[serde(default = "BluetoothConfig::default_format")]
     pub format: Option<FormatTemplate>,
+    #[serde(default = "BluetoothConfig::default_format_unavailable")]
```

Comment:
By the way, `Option<T>` implements Default that returns `None`, so no need to to create an explicit function for that. `#[serde(default)]` should work.

Reply:
Same for `bool`

Reply:
Oh neat, thanks!